.gitignore

```diff
@@ -1,3 +1,5 @@
 vendor/bundle
 .bundle
 Gemfile.lock
+# SSHFS
+._*
```

Comment:
Please do http://egorsmirnov.me/2015/05/04/global-gitignore-file.html instead of adding ._ to the gitignore of each project :-). Let me know if that will work.

Reply:
Done.